src/ResourceManager/Security/Commands.Security/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.Management.SecurityCenter" version="0.9.0-preview" targetFramework="net472" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.12" targetFramework="net472" />
```

Comment:
I got error messages trying to remove the package properly:
`Unable to uninstall 'Microsoft.Rest.ClientRuntime.Azure.3.3.13' because 'Microsoft.Azure.Management.SecurityCenter.0.9.0-preview' depends on it.`
`Unable to uninstall 'Microsoft.Rest.ClientRuntime.2.3.12' because 'Microsoft.Azure.Management.SecurityCenter.0.9.0-preview, Microsoft.Rest.ClientRuntime.Azure.3.3.13' depend on it.`
Unable to uninstall 'Newtonsoft.Json.6.0.8' because 'Microsoft.Azure.Management.SecurityCenter.0.9.0-preview, Microsoft.Rest.ClientRuntime.2.3.12, Microsoft.Rest.ClientRuntime.Azure.3.3.13' depend on it.

not sure it will be best to remove them manually from the packages.config
WDYT?